libcaja-private/caja-file.c

```diff
@@ -2555,6 +2555,25 @@ caja_file_update_info (CajaFile *file,
 	return update_info_internal (file, info, FALSE);
 }
 
+void
+caja_file_refresh_info (CajaFile *file)
+{
+	GFile *gfile;
+	GError *error;
+	GFileInfo *new_info;
+
+	gfile = caja_file_get_location (file);
+	new_info = g_file_query_info (gfile, CAJA_FILE_DEFAULT_ATTRIBUTES,
+	                              G_FILE_QUERY_INFO_NONE, NULL, &error);
```

Comment:
If *not* keeping `error`:
```suggestion
	                              G_FILE_QUERY_INFO_NONE, NULL, NULL);
```